2025/02/15/post-mortem.md

```diff
@@ -0,0 +1,194 @@
+# Post-Mortem: Network Outage Affecting OpenStreetMap.org - 15 December 2024
+
```

Comment:
```suggestion

All times are provided in Coordinated Universal Time (UTC).

```

Are only the times with "(UTC)" in UTC, or are "all times in UTC"? (if so, you can remove it elsewhere)